src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -75,5 +80,5 @@ abstract contract DelegationManagerStorage is IDelegationManager {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[44] private __gap;
+    uint256[43] private __gap;
```

Comment:
immutable variables don't occupy storage slots.
I only see a single immutable variable getting added (none removed, no others added), so it seems like this change would ruin the storage gap and potentially corrupt storage in an upgrade!

Reply:
fuck no this was because I added some storage and then deleted and forgot to update the gap uggh.  Good catch